BSPConvert.Lib/Source/MaterialConverter.cs

```diff
@@ -313,6 +315,8 @@ private static void AppendProxyVars(StringBuilder sb, ShaderStage texModStage)
 				}
 				else if (texModInfo.type == TexMod.TMOD_SCROLL)
 					sb.AppendLine("\t$translate \"[0.0 0.0]\"");
+				else if (texModInfo.type == TexMod.TMOD_SCALE)
+					sb.AppendLine($"\t$scale \"[{texModInfo.scale[0]} {texModInfo.scale[1]}]\"");
```

Comment:
Is this an animated tex mod? If not you should be able to use `$basetexturetransform` instead: https://developer.valvesoftware.com/wiki/$basetexture#.24basetexturetransform